game/static/game/css/lesson.css

```
@@ -0,0 +1,311 @@
+
+.lesson-board-wrapper{
+    display:flex;
+    justify-content:center;
+    margin:20px 0;
+}
+
+.lesson-board{
+    width:480px;
+    height:480px;
+
+    display:grid;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)